db/migrate/20150107233846_create_oauth_access_grants.rb

```diff
@@ -0,0 +1,15 @@
+class CreateOauthAccessGrants < ActiveRecord::Migration
```

Comment:
No I think I am just going to get rid of the gem/migrations. Related ticket here: https://github.com/neopoly/unschema/issues/2

It doesn't really add anything to the current branch at the moment although I could see it being used later on.